include/Options/Business/Nav_Menus.php

```diff
@@ -15,6 +17,14 @@
  * Class defining navigation menus array option.
  *
  * @since 3.7
+ *
+ * @phpstan-type Value array<
```

Comment:
```suggestion
 * @phpstan-type NavMenusValue array<
```
I found it to be too generic.
Also, should you do that for Domains as well? (I saw you had to add inline PHPStan doc as well).

Reply:
At the class level I don't think it's too generic, it's the type of the option the class handles. Other classes using it can alias it with `@phpstan-type Value from Nav_Menus as NavMenusValue`. But OK, if someday we need it outside this class, using a more specific name will prevent us to use the alias notation.

Reply:
Done in [d89ae43](https://github.com/polylang/polylang/pull/1610/commits/d89ae43bee89c9828ab8cd0f3084778b8285029f).

Reply:
Done in [0f3c69b](https://github.com/polylang/polylang/pull/1610/commits/0f3c69b36f857e56af03748af9e52e288afcf0b5) for `Domains`. I didn't do it at first because it isn't a complex type, and it isn't used outside the class.